src/main/java/com/revature/services/TestService.java

```diff
@@ -0,0 +1,14 @@
+package com.revature.services;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class TestService {
+	
+	public void breakBuild() {
+		AtomicInteger aInt1 = new AtomicInteger(0);
+		AtomicInteger aInt2 = new AtomicInteger(0);
+
+		if (aInt1.equals(aInt2)) { }  // Noncompliant
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Use ".get()" to retrieve the value and compare it instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS2204)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS00108)